### PR TITLE
phan: Add support for '--load-baseline'

### PR DESCRIPTION
**Pull request by kalkin**
| Q             | A
| ------------- | ---
| Branch        | master for features and deprecations
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Documented?   | yes

Phan supports suppressing known issues. This PR adds support for that to grumphp

### CODE REVIEW

**Comment by kalkin** [comment]
The failing appveyor has nothing to do with Phan. I think you just need to trigger the build again, but no idea how to do it.

**Comment by kalkin** [comment]
Bump?

File: test/Unit/Task/PhanTest.php [hunk 1]
```diff
@@ -27,6 +27,7 @@ public function provideConfigurableOptions(): iterable
             [
                 'output_mode' => 'text',
                 'output' => null,
+                'baseline' => '',
```
Comment (veewee) [inline test/Unit/Task/PhanTest.php:30]: Can you add a case for baseline in `provideExternalTaskRuns` as well?

**Comment by veewee** [comment]
Hello @kalkin ,

Thanks for the PR!
I'm a bit short on time currently. No worries, it will get merged in soon-ish :)

File: test/Unit/Task/PhanTest.php (hunk 1 above)
Comment (kalkin) [reply test/Unit/Task/PhanTest.php:30]: I do not fully understand the purpose of the `provideExternalTaskRuns` tests. Can you give me a hint how I implement it properly?

Comment (veewee) [reply test/Unit/Task/PhanTest.php:30]: The provider is added to validate that given the task config (first argument), the command line tool is being called with the correct CLI arguments (last argument).

**Comment by kalkin** [comment]
Sorry for not fixing this PR, but I found a workaround and have no time for it. Just add the `baseline_path` key to your `.phan/config.php` with the path to your baseline file.

**Pull request closed without merging**